### PR TITLE
add unique token filter docs #8451

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
add unique token filter docs

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/8451

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas @varun-lodaya Please provide tech review approval to move this PR forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

File: _analyzers/token-filters/index.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Normalization | `arabic_normalization`: [ArabicNormalizer](https://lucene.apache
 `synonym_graph` | N/A | Supplies a synonym list, including multiword synonyms, for the analysis process.
 `trim` | [TrimFilter](https://lucene.apache.org/core/9_10_0/analysis/common/org/apache/lucene/analysis/miscellaneous/TrimFilter.html) | Trims leading and trailing white space from each token in a stream. 
 `truncate` | [TruncateTokenFilter](https://lucene.apache.org/core/9_10_0/analysis/common/org/apache/lucene/analysis/miscellaneous/TruncateTokenFilter.html) | Truncates tokens whose length exceeds the specified character limit. 
-`unique` | N/A | Ensures each token is unique by removing duplicate tokens from a stream. 
+[`unique`]({{site.url}}{{site.baseurl}}/analyzers/token-filters/unique/) | N/A | Ensures each token is unique by removing duplicate tokens from a stream.
```
Comment (natebower) [inline _analyzers/token-filters/index.md:62]:
```suggestion
[`unique`]({{site.url}}{{site.baseurl}}/analyzers/token-filters/unique/) | N/A | Ensures that each token is unique by removing duplicate tokens from a stream. 
```

File: _analyzers/token-filters/unique.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Unique
+parent: Token filters
+nav_order: 450
+---
+
+# Unique token filter
+
+The `unique` token filter ensures that only unique tokens are kept during the analysis process, removing duplicate tokens that appear within a single field or text block.
```
Comment (natebower) [inline _analyzers/token-filters/unique.md:10]: "kept" => "retained"?

**Review by natebower — COMMENTED** [review]
@kolchfa-aws One comment and one change. Thanks!

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**